_hackbars/fernanda_costa.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: hackbar
+title: Open Source
+author: Fernanda Costa
+---
+
+Getting started to contribute to open source. Thanks for providing this opportunity.
+---
+
+Humans are polar bears only predator. Stop global warming. Save the polar bears.
```

Comment:
hey @fernanda-costa Can you change the line - 
```suggestion
## Humans are polar bears' only predator. Stop global warming. Save the polar bears.
```

Reply:
Sure :) sry

Reply:
it's okay